lib/code_fund/factory.ex

```diff
@@ -51,6 +51,7 @@ defmodule CodeFund.Factory do
   def creative_factory do
     %CodeFund.Schema.Creative{
       name: "Test Creative",
+      title: "Creative Title",
```

Comment:
difference between this and name?

Reply:
Name is what appears in the list, "Title" is what appears on the ad

Reply:
Any reason why they can't be the same?

Reply:
Title in this case will usually be the company name. For example, we can now have ads that look like this:

`{{title}} - {{description}}`

We don't want to use the name here b/c titles will most often be the same

Reply:
Can it be named something more descriptive that case, "display name" or "company name"?

Reply:
Just passing through ! I sort of confused when I saw `name` and `title` 😅